Formula/lablgtk.rb

```diff
@@ -3,6 +3,7 @@ class Lablgtk < Formula
   homepage "http://lablgtk.forge.ocamlcore.org"
   url "https://github.com/garrigue/lablgtk/archive/2.18.11.tar.gz"
   sha256 "ff3c551df4e220b0c0fb9a3da6429413bff14f8fc93f4dd6807a35463982c863"
+  license "LGPL-2.1"
```

Comment:
Isn't this `LGPL-2.0`? https://github.com/garrigue/lablgtk/blob/lablgtk3/LICENSE

Reply:
It's `LGPL-2.1`, at least according to https://github.com/garrigue/lablgtk/blob/lablgtk3/LGPL

Reply:
Hmm, it looks like you're right. The `LICENSE` file says `Library GPL` (which is `LGPL-2.0`) but then points to the `LGPL` file which has the text for `Lesser GPL` (`LGPL-2.1`).

I'm fine with `LGPL-2.1`

Reply:
the key diff between LGPL-2.1 and LGPL-2.0 is that `2.1`.
```
[This is the first released version of the Lesser GPL.  It also counts
 as the successor of the GNU Library Public License, version 2, hence
 the version number 2.1.]
```

Reply:
According to the [SPDX license list](https://spdx.org/licenses/), `Library General Public License`'s identifier is `LGPL-2.0`.

[The `LICENSE` file](https://github.com/garrigue/lablgtk/blob/65b109ce2ada0458d1716a37d6d60fbc645fc614/LICENSE#L1-L3) specifically says `Library General Public License`, which could indicate `LGPL-2.0`.

The [`LGPL` file](https://github.com/garrigue/lablgtk/blob/lablgtk3/LGPL) says `Lesser GPL` (`LGPL-2.1`). There's some inconsistency among the two license files in the `lablgtk` repo which is why I'm checking.

Reply:
But because the `LGPL-2.1` text is included, I think it's the right one to go with.

Reply:
ok, I see where you come from now. yeah, I will PR to fix that unclearness. :)

Reply:
I think it is actually good LGPL exception. (check this out, https://github.com/andrenth/ocaml-fd/blob/master/debian/copyright#L11-L27)

The software is licensed under LGPL 2.1, but the license exception is applicable to LGPL 2.0

Reply:
Does this need to be indicated in the formula or is just having `LGPL-2.1` listed enough?